Config/Project64.rdb

```diff
@@ -150,6 +150,15 @@ Bad ROM?.AutoFullScreen=False
 
 // ---- END OF HEADER, START OF ROM ENTRIES --------------------------
 //================  0-9  ================
+[B2242748-FFBD61DA-C:45]
+Good Name=007 - Goldfinger (Beta)
+32bit=No
+Counter Factor=1
+Fast SP=0
+Good Name=GOLDFINGER
+Linking=0
+RDRamSize=8388608
```

Comment:
RDB has different syntax compared to .cfg.
RDRAM Size=8
Linking=Off

Reply:
Fixed it. Thanks.